tools/cellxgene_census_builder/src/cellxgene_census_builder/build_soma/experiment_builder.py

```diff
@@ -361,8 +361,11 @@ def write_X_normalized(self, args: CensusBuildArgs) -> None:
             return
 
         logging.info(f"Write X normalized: {self.name} - starting")
-        # grab the previously calculated sum of the X['raw'] layer
-        raw_sum = self.obs_df.raw_sum.to_numpy()
+        is_smart_seq = np.isin(
+            self.obs_df.assay_ontology_term_id.to_numpy(), ["EFO:0008930", "EFO:0008931", "EFO:0700016"]
```

Comment:
These are already defined as a `SMART_SEQ` constant in the code below - use that instead?

Reply:
yes, good catch.  fixing.